File: js/src/1. Big-O/findNemo.js
const nemo = ["memo"];
const largeArray = new Array(100).fill('nemo');

const findNemo = (array) => {
  let t0 = performance.now();
  for (let i = 0; i < array.length; i++) {
    if (array[i] === "nemo") {
      console.log("Found Nemo!");
    } else {
      console.log("NEMO not found");
    }
  }
  let t1 = performance.now();
  console.log('Time == ' + (t1 - t0) + ' ms');
};

findNemo(largeArray); // O(n)

File: python/src/1. Big-O/find_nemo.py
import time

nemo = ["nemo"]
large_array = ["nemo"] * 100


def find_nemo(array):
    t0 = time.time()
    for item in array:
        if item == "nemo":
            print("Found NEMO")
        else:
            print("Nemo not Found!")

    t1 = time.time()
    print("Time taken ==", (t1 - t0), "Ms")


find_nemo(large_array)  # O(n)
